### PR TITLE
Fix/remove blockquote

### PR DESCRIPTION
**Pull request by jthrilly**
- [x] Removes block quote support. Angle brackets are now no longer treated as shortcuts for block quote - they are serialised as HTML entities
- [x] Fixed issue where editor would think thematic break was an empty element
- [x] Removed the markdown shortcut system, and added escaping for markdown characters. This dramatically simplifies normalization and removes almost all edge cases.

### CODE REVIEW

**Review by wwqrd — COMMENTED** [review]
Just code review at the moment. Looks good. Think we could delete the shortcuts file now?

**Comment by wwqrd** [comment]
For me this seems to work, but I still can't reproduce the crashing error.

File: src/components/Fields/RichText/lib/serialize.js [hunk 1]
```diff
@@ -1,10 +1,38 @@
 import { serialize } from 'remark-slate';
 
+const escapeMarkdownChars = (string) => string
+  .replace(/\\/g, '\\\\')
+  .replace(/\*/g, '\\*')
+  .replace(/_/g, '\\_')
```
[the remark this answers is not in the document]
Comment (jthrilly) [reply src/components/Fields/RichText/lib/serialize.js:7]: I'd like to keep them separate for readability. Possibly less good for you, but it helps out novices like me.

File: src/components/Fields/RichText/RichText.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ import isHotkey from 'is-hotkey';
 import { compose, isEmpty } from 'lodash/fp';
 import { EditListPlugin } from '@productboard/slate-edit-list';
 import withNormalize from './lib/withNormalize';
-import withShortcuts from './lib/withShortcuts';
+// import withShortcuts from './lib/withShortcuts';
```
[the remark this answers is not in the document]
Comment (jthrilly) [reply src/components/Fields/RichText/RichText.js]: done

**Pull request merged**